android/app/src/main/kotlin/com/chicagoroboto/features/main/MainActivity.kt

```diff
@@ -40,7 +39,7 @@ class MainActivity : AppCompatActivity(), SessionNavigator, SpeakerNavigator, Na
             it.setDisplayHomeAsUpEnabled(true)
         }
 
-        setTitle(R.string.event_name)
+        setTitle(R.string.action_speakers)
```

Comment:
This should default to the event name (currently, launching the app shows "Speakers" as the title.

Reply:
I'll remove this `setTitle` from onCreate and have the `showView` method, which pulls the toolbar title from the view set it instead. That way, whenever we call `showView()` with any view, the correct title will be displayed.